docs/components/plugins/PaddedAPIBox.tsx

```diff
@@ -0,0 +1,13 @@
+import { PropsWithChildren } from 'react';
+
+import { APIBox } from '~/components/plugins/APIBox';
+
+type PaddedAPIBoxProps = PropsWithChildren<unknown>;
+
+export function PaddedAPIBox({ children, ...props }: PaddedAPIBoxProps) {
+  return (
+    <APIBox className="px-4 py-3" {...props}>
```

Comment:
```suggestion
    <APIBox className={mergeClasses("px-4 py-3", className)} {...props}>
```

Since we will be using `APIBoxProps` class name would be a valid prop, so let's make sure the passed value is not lost.